common/models/change.js

```diff
@@ -96,7 +99,9 @@ module.exports = function(Change) {
     });
 
     async.parallel(tasks, function(err) {
-      if (err) return callback(err);
+      if (err) {
+        callback(err);
+      }
```

Comment:
Actually the return is important here! If you don't return from the function, then the code below is run even when there was an error.